webapp-django/crashstats/crashstats/jinja2/crashstats_base.html

```diff
@@ -55,17 +55,17 @@
     <h1>Product Navigation</h1>
 
     <div class="version-nav">
-      <input type="hidden" id="url_base" name="url_base" value="/products/{{ product }}" />
-      <input type="hidden" id="url_site" name="url_site" value="/" />
-
       {% block product_nav_filter %}
+        <input type="hidden" id="url_base" name="url_base" value="/products/{{ product.name }}" />
+        <input type="hidden" id="url_site" name="url_site" value="/" />
+
         <span class="quick-nav-title">Quick Navigation</span>
         <ul class="filter">
           <li>
             <label for="products_select" class="visually-hidden">Select Product</label>
             <select id="products_select">
-              {% for product_name in products %}
-                <option value="{{ product_name }}" {% if product_name == product %}selected{% endif %}>{{ product_name }}</option>
+              {% for p in products %}
```

Comment:
"products" is now a list of Product instances and "product" is a Product instance in the context.